patches/ima-no-cache-v3/0000-cover-letter.patch

```diff
@@ -0,0 +1,41 @@
+From 9e559a7b6a2b4cac1c5e458872619721d78158d3 Mon Sep 17 00:00:00 2001
+From: Alban Crequy <alban@kinvolk.io>
+Date: Fri, 19 Jan 2018 12:40:11 +0100
+Subject: [RFC PATCH v3 0/2] ima,fuse: introduce new fs flag FS_IMA_NO_CACHE
+
+This patchset v3 introduces a new fs flag FS_IMA_NO_CACHE and uses it in
+FUSE. This forces files to be re-measured, re-appraised and re-audited
+on file systems with the feature flag FS_IMA_NO_CACHE. In that way,
+cached integrity results won't be used.
+
+There was a previous attempt (unmerged) with a IMA option named "force" and using
+that option for FUSE filesystems. This patches use a different approach
```

Comment:
Either `These patches use` or `This patchset uses`?

Reply:
updated

Reply:
Looks good!